packages/dds/tree/package.json

```diff
@@ -31,8 +31,11 @@
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
     "fence": "gf",
-    "lint": "npm run eslint",
-    "lint:fix": "npm run eslint:fix",
+    "format": "npm run prettier:fix",
+    "lint": "npm run prettier && npm run eslint",
```

Comment:
if you are trying to just add support for running prettier in this PR, but not yet require everything is formatted with it, you should not be changing the lint command to use it yet.